Game version guessing fails to find resource_packs

[//]: # (Thank you for taking the time to report an issue or propose an enhancement. Questions are also welcome.)
[//]: # (If you have more than one item to report, please open separate issues!)

[//]: # (This text will not be visible and you can freely delete it. Check the Preview tab for a cleaner look.)

### What is wrong

When I click the guess game version button in the launcher, I get the failure dialog. In the logs it seems like it cannot find the resource_packs directory. The resource_packs themselves are installed by the deb package. It seems like the launcher is using `externalized/resource_packs` instead of `{EXTRA_DATA_DIR}/externalized/resource_packs`. Imho this should work for `0.17.0`, so I added the respective milestone.

```
2020-07-20T17:17:31 [ INFO] stracciatella::guess: Looking for data dir in "/home/ubuntu"
2020-07-20T17:17:31 [ INFO] stracciatella::guess: Found "/home/ubuntu/data"
2020-07-20T17:17:31 [ INFO] stracciatella::guess: Getting resources with_archive_slf
2020-07-20T17:17:32 [ INFO] stracciatella::guess: Inspecting resource paths for language specific files
2020-07-20T17:17:32 [ INFO] stracciatella::guess: Number of language specific files: 0 Dutch, 0 German, 0 Italian, 0 Polish, 0 Russian
2020-07-20T17:17:32 [ INFO] stracciatella::guess: Detected no language specific files or multiple different languages
2020-07-20T17:17:32 [ INFO] stracciatella::guess: Searching for resource packs in "externalized/resource_packs"
2020-07-20T17:17:32 [ERROR] stracciatella::guess: Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

[//]: # (For bugs, please fill in this table [otherwise delete it]. The lines don't have to stay aligned.)

| JA2S version | Game version        | Vanilla bug? | Used to work? |
| ------------ | ------------------- | ------------ | ------------- |
|  0.17.0-rc | -  |    -    |    yes     |

```
```

[//]: # (Please also include:)
[//]: # (- how to reproduce the bug and, if possible, a save game)
[//]: # (- the operating system)
[//]: # (- in case you know, for enhancements, if this feature was already added by some other source mod like 1.13)

[//]: # (When pasting backtraces or any other logs with hashed enumerations, please format it as a code block.)
[//]: # (You can do that by selecting it and clicking the "<>" icon or manually putting three backticks [```] at each end.)

--- a/rust/stracciatella/src/guess.rs
+++ b/rust/stracciatella/src/guess.rs
@@ -16,6 +16,7 @@ use serde_json;
 
 use crate::config::VanillaVersion;
 use crate::fs::resolve_existing_components;
+use crate::get_assets_dir;
 use crate::res::{
     Resource, ResourceError, ResourcePack, ResourcePackBuilder, ResourcePropertiesExt,
 };
@@ -272,7 +273,12 @@ impl Guess {
 
     /// Find all resource packs in externalized directory
     fn get_pack_paths(&self) -> GuessResult<Vec<PathBuf>> {
-        let dir = Path::new("externalized/resource_packs");
+        let asset_dir = get_assets_dir();
+        let dir = resolve_existing_components(
+            Path::new("externalized/resource_packs"),
+            Some(&asset_dir),
+            true,
+        );
         info!("Searching for resource packs in {:?}", &dir);
         let paths: Vec<PathBuf> = dir
             .read_dir()?

